skaffold.yaml

```diff
@@ -9,40 +20,95 @@ build:
-          context: deployments/Dockerfiles/autonomy
-          docker:
-            noCache: true
-            dockerfile: Dockerfile
-            buildArgs:
-              AEA_VERSION: "1.17.0"
-
-        - image: valory/open-autonomy-tendermint
-          context: deployments/Dockerfiles/tendermint/
-          docker:
-            noCache: true
-            dockerfile: Dockerfile
-
-        - image: valory/open-autonomy-hardhat
-          context: deployments/Dockerfiles/hardhat
-          docker:
-            noCache: true
-
-        - image: valory/open-autonomy-docs
-          docker:
-            noCache: true
-            dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+      tagPolicy:
+        envTemplate:
+          template: "{{.VERSION}}"
+      artifacts:
+
+      - image: valory/open-autonomy
+        context: deployments/Dockerfiles/autonomy
+        docker:
+          <<: *release_docker_args
+          buildArgs: 
+            <<: *aea_version
+
+      - image: valory/open-autonomy-tendermint
+        context: deployments/Dockerfiles/tendermint/
+        docker:
+          <<: *release_docker_args
+
+      - image: valory/open-autonomy-hardhat
+        context: deployments/Dockerfiles/hardhat
+        docker:
+          <<: *release_docker_args
+
+      - image: valory/open-autonomy-docs
+        docker:
+          <<: *release_docker_args
+          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+
+      - image: valory/oar-oracle
+        <<: *open_autonomy_runtime_context
+        docker:
+          <<: *release_docker_args
+          buildArgs: &oar-oracle
+            <<: *aea_version
+            AEA_AGENT: "{{.AEA_AGENT_ORACLE}}"
+
+      - image: valory/oar-apy_estimation
+        <<: *open_autonomy_runtime_context
+        docker:
+          <<: *release_docker_args
+          buildArgs: &oar-apy_estimation
+            <<: *aea_version
+            AEA_AGENT: "{{.AEA_AGENT_APY_ESTIMATION}}"
+
+  - name: release-latest
+    build:
+      tagPolicy:
+        envTemplate:
+          template: latest
+      artifacts:
+
+      - image: valory/open-autonomy
+        context: deployments/Dockerfiles/autonomy
+        docker:
+          <<: *docker_args
+          buildArgs: 
+            <<: *aea_version
+
+      - image: valory/open-autonomy-tendermint
+        context: deployments/Dockerfiles/tendermint/
+        docker:
+          <<: *docker_args
+
+      - image: valory/open-autonomy-hardhat
+        context: deployments/Dockerfiles/hardhat
+        docker:
+          <<: *docker_args
+
+      - image: valory/open-autonomy-docs
+        docker:
+          <<: *docker_args
+          dockerfile: deployments/Dockerfiles/documentation/Dockerfile
+
+      - image: valory/oar-oracle
+        <<: *open_autonomy_runtime_context
+        docker:
+          <<: *docker_args
+          buildArgs:
+            <<: *oar-oracle
+
+      - image: valory/oar-apy_estimation
```

Comment:
```suggestion
      - image: valory/oar-apy-estimation
```

Reply:
We need to keep this `apy_estimation` because this is supposed to be the name of the agent, in this case it's `apy_estimation`